README.md

```diff
@@ -19,60 +19,45 @@ yarn install
 
 Client for StreamDB uses generated gRPC stubs to provide fast access to data. 
 
-#### Documentation 
-Link to documentation can be found here.
-[https://buf.build/proximaone/streamapis/docs/main/messages.v1alpha1](https://buf.build/proximaone/streamapis/docs/main/messages.v1alpha1)
-
-
 ### Sample 
 
 Check out full solution at https://github.com/proxima-one/proxima-samples/tree/master/eth-blockheaders-stream
 
 ```typescript 
```

Comment:
How about leaving only eye-catchy examples of how easy it is to consume the stream in the readme? And moving these detailed examples which show more functionality and details in a separate folder